test/inductor/test_select_algorithm.py

```diff
@@ -369,6 +371,40 @@ def test_TritonTemplateCaller_str(self):
         self.assertEqual(caller_str, f"TritonTemplateCaller({module_path}, extra)")
 
 
+def make_benchmark_fn(
+    cls,
+    _,
+    input_nodes,
+    layout,
+    input_gen_fns,
+):
+    def benchmark(choices):
+        request = RemoteBenchmarkRequest.from_choices(choices, input_nodes, layout)
+        request = pickle.loads(pickle.dumps(request))
+        timings = request.benchmark()
+        return dict(zip(choices, timings))
+
+    return benchmark
+
+
+class TestSelectAlgorithmRemote(TestSelectAlgorithm):
```

Comment:
nice ! it might be worth doing this on a subset of the test_max_autotune tests as well